disnake/ext/commands/base_core.py

```diff
@@ -140,8 +142,14 @@ class InvokableApplicationCommand(ABC):
         .. versionadded: 2.5
     """
 
+    __original_kwargs__: Dict[str, Any]
     body: ApplicationCommand
 
+    def __new__(cls, *args: Any, **kwargs: Any) -> InvokableApplicationCommand:
+        self = super().__new__(cls)
+        self.__original_kwargs__ = {k: v for k, v in kwargs.items() if v is not None}
```

Comment:
why do `if v is not None`?

Reply:
Oh.

Reply:
https://github.com/DisnakeDev/disnake/pull/501/commits/1669c1889c214c1ceaab89ead94657d7099dd82f